src/PackGlobalTool.targets

```diff
@@ -12,7 +12,7 @@
         description=$(Description);
         repositoryUrl=$(RepositoryUrl);
         targetframework=$(TargetFramework);
-        MicrosoftNETCorePlatformsPackageVersion=$(MicrosoftNETCorePlatformsPackageVersion);
+        repositoryCommit=$(RepositoryCommit);
```

Comment:
Is this a new thing that's needed?

Reply:
Yes, it's new feature of NuGet we started using last week. It'ss unrelated to #400, but figured I'd fix it as long as I'm editing the nuspec.